appeals/web/src/server/appeals/appeal-details/appellant-case/__tests__/__snapshots__/appellant-case.test.js.snap

```diff
@@ -1224,6 +1224,68 @@ exports[`appellant-case POST /appellant-case/incomplete/date should re-render th
 </main>"
 `;
 
+exports[`appellant-case POST /appellant-case/incomplete/date should re-render the update date page with the expected error message if an invalid day was provided 4`] = `
+"<main class="govuk-main-wrapper " id="main-content" role="main">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <div class="govuk-error-summary" data-module="govuk-error-summary">
+                <div role="alert">
+                    <h2 class="govuk-error-summary__title"> There is a problem</h2>
+                    <div class="govuk-error-summary__body">
+                        <ul class="govuk-list govuk-error-summary__list">
+                            <li><a href="#">Date must be a date in the future</a>
```

Comment:
This doesn't look like the expected error message for this test, I think something may be wrong in the validator

Reply:
Actually it looks like you might just need to update the dates in some of the unit tests - some of the test dates are in the past, which wasn't an issue previously. It would be worth checking the updated snapshots once that's done to make sure everything is working as expected